src/psyclone/psyir/symbols/__init__.py

```diff
@@ -33,32 +33,35 @@
 # -----------------------------------------------------------------------------
 # Author S. Siso, STFC Daresbury Lab
 # Modified by A. R. Porter and R. W. Ford, STFC Daresbury Lab
+# Modified by J. Henrichs, Bureau of Meteorology
 # -----------------------------------------------------------------------------
 
 ''' Symbols package module '''
 
 from psyclone.psyir.symbols.datasymbol import DataSymbol
 from psyclone.psyir.symbols.containersymbol import ContainerSymbol
 from psyclone.psyir.symbols.data_type_symbol import DataTypeSymbol
-from psyclone.psyir.symbols.interfaces import AutomaticInterface, \
-    ImportInterface, ArgumentInterface, UnresolvedInterface, StaticInterface, \
-    DefaultModuleInterface, UnknownInterface, CommonBlockInterface
+from psyclone.psyir.symbols.interfaces import \
+    (ArgumentInterface, AutomaticInterface, CommonBlockInterface,
+     DefaultModuleInterface, ImportInterface,
+     StaticInterface, UnknownInterface, UnresolvedInterface)
 from psyclone.psyir.symbols.intrinsic_symbol import IntrinsicSymbol
 from psyclone.psyir.symbols.routinesymbol import RoutineSymbol
 from psyclone.psyir.symbols.symbol import Symbol, SymbolError
 from psyclone.psyir.symbols.symbol_table import SymbolTable
 from psyclone.psyir.symbols.typed_symbol import TypedSymbol
-from psyclone.psyir.symbols.datatypes import DataType, ScalarType, \
-    ArrayType, TYPE_MAP_TO_PYTHON, REAL_TYPE, REAL_SINGLE_TYPE, \
-    REAL_DOUBLE_TYPE, REAL4_TYPE, REAL8_TYPE, INTEGER_TYPE, \
-    INTEGER_SINGLE_TYPE, INTEGER_DOUBLE_TYPE, INTEGER4_TYPE, INTEGER8_TYPE, \
-    BOOLEAN_TYPE, CHARACTER_TYPE, DeferredType, UnknownType, \
-    UnknownFortranType, StructureType, NoType
+from psyclone.psyir.symbols.datatypes import \
+    (ArrayType, BOOLEAN_TYPE, CHARACTER_TYPE, DeferredType, DataType,
```

Comment:
I think its a bit nicer to avoid the backslash by starting the parenthesis in the first line, then there can be the linebreak just after the parenthesis.

Reply:
Just to double check: if I add the parenthesis at the line above, then all following imported symbols must be aligned to the opening parenthesis, or else we get a pycodestyle error (continuation line under-indented). Given that the `from ... import` is already long, this means the 4 or 5 lines of import we have atm, become 12 lines. Similarly, the 3 lines above importing from interface become 8 lines.

TBH, I don't really care. I personally prefer to have the import statements more compact. So, unless you tell me to leave it, I'll fix this according to your review (tomorrow).

Reply:
I mean like in `src/psyclone/psyir/nodes/__init__.py`, it is just substituting the backslash by the opening parenthesis, all the imports are still at the same place as now.

Reply:
Thanks, I never realised that this was possible :)